packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```
@@ -3,6 +3,7 @@
 
 @use "../../functions" as *;
 @use "../../settings" as *;
+@use "../../variables" as *;
```

Do you need all of the variables? Alternatively you can bring in jus the high contrast mode border:

```suggestion
@use "../../variables/border-high-contrast" as *;
```

---

Resolved in [264008a](https://github.com/uswds/uswds/pull/5147/commits/264008a89c4728f12eece6c295435208b435e26f)